components/Alert/alert.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Alert</title>
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+    />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v6.0.0-beta3/css/all.css"
+    />
+    <link rel="stylesheet" href="../../css/winsome.css" />
+  </head>
+  <body>
+    <div class="grid-container">
+      <header class="header-bar">
+        <div class="header-item">
+          <!-- <div class="header-logo">
+            <img src="../../assets/logo.png" alt="logo" />
+          </div> -->
```

Comment:
remove commented code

Reply:
Thanks Shraddha for noticing it, I have removed the comments now.
For iFrame I am not seeing the default carbon code, can you attach screenshot.

Reply:
It was showing in the morning, maybe issue with carbon sometimes. 
Looks good now.